### ISSUE TITLE
Video playback speed

### ISSUE THREAD

**Issue by mizzunet** [LagradOst/CloudStream-2#32]
It would be very handy to have the ability to change the speed of playback, don't you think?

**Comment by LagradOst** [comment]
Yes, however I never use this feature, and therefore it is not in the app. All development on this app was however stopped some time ago to work on cs-3, so this feature wont come to cs-2 but will be in cs-3. 😔

**Comment by mizzunet** [comment]
@LagradOst take your time. I'm very grateful that I found this app. Thank you so much for developing this awesome app :) Good luck
